cloud/disk_manager/internal/pkg/services/disks/migrate_disk_task.go

```diff
@@ -333,17 +333,6 @@ func (t *migrateDiskTask) scheduleReplicateTask(
 	execCtx tasks.ExecutionContext,
 ) error {
 
-	diskMeta, err := t.resourceStorage.GetDiskMeta(ctx, t.request.Disk.DiskId)
```

Comment:
Do we still need resourceStorage here?

Reply:
yup

https://github.com/ydb-platform/nbs/blob/68ce0e6c84cf3be0a1ac57db5d84491e8a96ee43/cloud/disk_manager/internal/pkg/services/disks/migrate_disk_task.go#L411

https://github.com/ydb-platform/nbs/blob/68ce0e6c84cf3be0a1ac57db5d84491e8a96ee43/cloud/disk_manager/internal/pkg/services/disks/migrate_disk_task.go#L499